### ISSUE TITLE
Conflict with Fontawesome 5 SVG with JS option

### ISSUE THREAD

**Issue by macsupport** [framework7io/framework7#2188]
This is a (multiple allowed):
* [x] bug
* [ ] enhancement
* [ ] feature-discussion (RFC)

* Framework7 Version: 2.0.7
* Platform and Target:  IOS, ANDROID
* Live Link or JSFiddle: PREFERABLY _(IF YOU WANT YOUR ISSUE TO BE RESOLVED ASAP)_. You can start from forking the following JSFiddle template: [JSFiddle](https://jsfiddle.net/nolimits4web/o2ejupu1/)

### What you did
When adding a Floating Action Button  `<div class="fab fab-right-bottom fab-morph" data-morph-to=".toolbar" style=""><a href="#" class=""><i class="icon f7-icons ios-only">add</i><i class="icon f7-icons ios-only">close</i></a></div>` the class "fab" conflicts with  fopntawesome 5 class of "fab" and the floating action button is converted to an svg by fontawesome 5's javascript. 

### Expected Behavior
What is expected is that the Floating Action Button will be rendered appropriately

### Actual Behavior
The Floating Action Button is converted by fontawesome5 svg with js javascript to an svg and the FAB is not rendered correctly.

P.S. Remember, an issue is not the place to ask questions. You can use [Stack Overflow](http://stackoverflow.com/questions/tagged/framework7)
for that or join the #framework7 channel on irc.freenode.net, where we will be more
than happy to help answer your questions.

Before you open an issue, please check if a similar issue already exists or has been closed before.

**Comment by JCKodel** [comment]
On https://github.com/FortAwesome/Font-Awesome/blob/c8166a2f65c3ca3cfada68b7614c51ef540f5c6f/advanced-options/use-with-node-js/fontawesome-free-webfonts/scss/_core.scss, change the prefix from fab to anything else.

It's far more easy for a simple icon library change some generated CSS than an entire framework change just because Font Awesome did a pretty lousy job conflicting their classes with a well-known Android feature, don't you think?

**Comment by macsupport** [comment]
I agree but I don’t actually use the fab icons (brands) or have the fab class in my project  . The issue occurs due to the JavaScript acting on the fab class.
